src/PAModel/Serializers/MsAppSerializer.cs

```diff
@@ -295,6 +298,25 @@ public static CanvasDocument Load(Stream streamToMsapp, ErrorContainer errors)
                 if (!string.IsNullOrEmpty(app._properties.LocalConnectionReferences))
                 {
                     var cxs = Utilities.JsonParse<IDictionary<String, ConnectionJson>>(app._properties.LocalConnectionReferences);
+
+                    foreach (var connectionJson in cxs)
+                    {
+                        var extensionData = connectionJson.Value.ExtensionData;
+                        if (extensionData != null)
+                        {
+                            if (extensionData.TryGetValue(ConnectionInstanceIDPropertyName, out JsonElement connectionInstanceID))
+                            {
+                                var serializedID = JsonSerializer.Serialize(connectionInstanceID);
```

Comment:
Since this is just a string value (as far as I can tell) do we need to JSON serialize/deserialize it?

Reply:
> Since this is just a string value (as far as I can tell) do we need to JSON serialize/deserialize it?
Good point. So I started without serialization and then found out it throws " invalid start to JSON with character '/' ".Basically since it stores something like a URL, decided to serialize and it works.